regolith/builders/reimbursementbuilder.py

```
@@ -55,103 +62,121 @@ def construct_global_ctx(self):
-                    dates.append(mdy_date(**item))
-                    item_ws.cell(row=r, column=2, value=i)
-                    item_ws.cell(row=r, column=3, value=mdy(**item))
-                    item_ws.cell(row=r, column=purpose_column, value=item["purpose"])
-                    item_ws.cell(
-                        row=r,
-                        column=ue_column,
-                        value=item.get("unsegregated_expense", 0),
+                        if r > 49:
+                            item_ws = wb["Extra_Page"]
+                            j = 0
+                            r = j + i
+                            purpose_column = 5
+                            ue_column = 12
+                            se_column = 14
+                        dates.append(mdy_date(**item))
+                        item_ws.cell(row=r, column=2, value=i)
+                        item_ws.cell(row=r, column=3, value=mdy(**item))
+                        item_ws.cell(row=r, column=purpose_column,
+                                     value=item["purpose"])
+                        item_ws.cell(
+                            row=r,
+                            column=ue_column,
+                            value=item.get("unsegregated_expense", 0),
+                        )
+                        try:
+                            total_amount += item.get("unsegregated_expense", 0)
+                        except TypeError:
+                            if item.get("unsegregated_expense", 0) == 'tbd':
+                                print("WARNING: unsegregated expense in {} is "
+                                      "tbd".format(ex["_id"]))
+                                item["unsegregated_expense"] = 0
+                            else:
+                                sys.exit(
+                                    "ERROR: unsegregated expense in {} is not "
+                                    "a number".format(ex["_id"]))
```

You should not use sys.exit in library code.  Please raise an approriate error instead